cmd/csaf_downloader/downloader.go

```diff
@@ -92,9 +92,17 @@ func (d *downloader) addStats(o *stats) {
 	d.stats.add(o)
 }
 
+// checkRedirect specifies the policy for handling redirects.
+func checkRedirect(req *http.Request, via []*http.Request) error {
+	slog.Debug(fmt.Sprintf("Redirecting to %s", req.URL))
```

Comment:
Would like to to see the use of the `s` for structured in `slog`.
Something like:
```
slog.Debug("Redirecting", "to", req.URL.String())
```